libraries/commonIO/src/main/java/org/eclipse/sw360/commonIO/AttachmentFrontendUtils.java

```diff
@@ -53,7 +53,8 @@ public InputStream getStreamToServeAFile(Collection<AttachmentContent> attachmen
         if(attachments == null || attachments.size() == 0){
             throw new SW360Exception("Tried to download empty set of Attachments");
         }else if(attachments.size() == 1){
-            return getConnector().getAttachmentStream(attachments.stream().findAny().get(), user, context);
+            // Temporary solutions, permission check needs to be implemented (getAttachmentStream)
+            return getConnector().unsafeGetAttachmentStream(attachments.iterator().next());
```

Comment:
This is an important change, since it has security implications, which is hidden as a one liner in this PR. I would find it better if
- it is expressed in the PR description and
- a corresponding issue is created.

Otherwise it will get lost.